spec/services/moderation_services/entourage_moderation_spec.rb

```diff
@@ -4,18 +4,19 @@
 
   describe '#.on_create' do
     subject { described_class.on_create(entourage) }
-    let(:entourage) { FactoryGirl.create(:entourage, updated_at: 10.hours.ago) }
+    let(:entourage) { create :entourage, updated_at: 10.hours.ago }
 
     context 'when admin exists' do
-      let!(:user) { create :admin_user }
+      let!(:admin) { create :admin_user }
```

Comment:
Vu que `admin` est maintenant utilisé a plusieurs endroits, vaut-il mieux le laisser tel quel ou le passer en lazy ? (Et du coup rajouter une ligne dans *should not change number_of_people*)